workspaces/github-pull-requests-board/plugins/github-pull-requests-board/src/hooks/useUserRepositoriesAndTeam.tsx

```diff
@@ -76,6 +84,6 @@ export function useUserRepositoriesAndTeam() {
     loading,
     repositories: teamData.repositories,
     teamMembers: teamData.teamMembers,
-    teamMembersOrganization: getGithubOrganizationFromEntity(teamEntity),
+    teamMembersOrganization: getGithubOrganizationFromEntity(ownerEntities[0]),
```

Comment:
I'm not familiar with this plugin, does this mean it is always derived from the first entity? Perhaps that is intentional?

Reply:
thanks, yes is intended, that field is used on the current EntityPage, that has always only 1 entity.

in the future if multiple organization tracking is needed i think it would be better to make a bigger refactor to decouple it a bit more, but as it now I tried to no go full down in the rabbit hole